scripts/train.py

```diff
@@ -0,0 +1,41 @@
+import os
+import click
+import pickle
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--PKSpace', 'dataset_mode', flag_value='PKSpace',
+              default=True)
+@click.option('--PKLot', 'dataset_mode', flag_value='PKLot')
+@click.option('--MLP', 'method', flag_value='MLP',
+              default=True, help='Method to be used for prediction')
+@click.option('--dataset_dir', required=True,
+              help='Directory of dataset for model to be trained on')
+@click.option('--output', default=None,
+              help='Name of output file for trained model')
```

Comment:
@rstevanak Here, we should probably mentioned the default name of the model if there is nothing specified by the user, i.e., `out.pkl`. So probably should be written in default argument.